docs/glossary.rst

```diff
@@ -63,20 +63,20 @@ Azure AD, refer to the `Microsoft documentation on Azure`_.
 Cluster
 -------
 
-Within each :ref:`project <gloss-project>`, a project administrator can deploy
-any number of :ref:`products <gloss-product>`. One such service is the
-deployment of clusters, which can be done through the CrateDB Cloud Console. A
-cluster is a set of two or more CrateDB instances (referred to as nodes) which
-form a single, distributed database. Effectively, each cluster within CrateDB
-Cloud is a hosted (part of) a database. Depending on the user's
+Within each :ref:`organization <gloss-org>`, an administrator can deploy any
```

Comment:
We might want to clarify that for the purpose of CrateDB Cloud, we also treat single-node clusters as clusters (which are useful for testing purposes). Not sure how to phrase it though.

Reply:
Good catch, I missed that. Yeah the 2+ comment is interesting, not sure where that came from. Hopefully clearer now, thanks.